WBooks/Book Comments/BookCommentsView.swift

```diff
@@ -34,15 +36,19 @@ class BookCommentsView: UIView, NibLoadable {
         loadingCommentsIndicator.stopAnimating()
     }
     
-    func displayNoCommentsYet() {
-        let noComments = UILabel()
-        noComments.text = "NO_COMMENTS".localized()
-        noComments.textColor = UIColor.gray
-        self.addSubview(noComments)
-        noComments.translatesAutoresizingMaskIntoConstraints = false
+    func displayCommentsIndicator(state: TableState) {
+        if state == TableState.empty {
+            commentIndicator.text = "NO_COMMENTS".localized()
+        }
+        if state == TableState.withValues {
+            commentIndicator.text = "COMMENT_ERROR".localized()
```

Comment:
if the state is `withValues` why are you setting an error message?

Reply:
Typo right there, my bad.